app/views/insured/product_shoppings/show.html.slim

```diff
@@ -377,8 +387,13 @@ javascript:
       `
       <div class="module plan-row panel panel-default" data-cuke="filtered-plan">
               <div class="panel-body">
-                <div class="row ${result.product.is_standard_plan? '' : 'hidden'}">
-                  <span class="ml-1 text-uppercase" style="color:green"><i class="fas fa-bookmark"></i> Standard Plan</span>
+                <div class="row">
+                  <span class=" ${result.is_pvp_eligible? '' : 'hidden'} pvp-plan-indicator ml-1">
+                    <i class="fas fa-lg fa-square"></i> Premium Value Plan
```

Comment:
are these meant to omit tooltip info?

Reply:
This seems like a bug. Personally, my impulse is to fix it while we're in here, and avoid it potentially coming out in UAT and having to revisit under more pressure, but up to you.

Reply:
nice! thanks!